.rubocop.yml

```diff
@@ -9,3 +9,4 @@ AllCops:
   Exclude:
     - 'spec/dummy/db/**/*'
     - 'vendor/**/*'
+    - 'gemfiles/vendor/**/*'
```

Comment:
we shouldn't need this once we update our CI install command to put these files in the right place